cmake/modules/libelf.cmake

```diff
@@ -13,6 +13,9 @@ elseif(NOT USE_BUNDLED_LIBELF)
     else()
         message(FATAL_ERROR "Couldn't find system libelf")
     endif()
+    # We add a custom target, in this way we can always depend on `libelf`
+    # without distinguishing between "bundled" and "not-bundled" case
+    add_custom_target(libelf)
```

Comment:
This is neat! We might get rid of all the `if (USE_BUNDLE_FOO) add_dependencies(... foo)`, right?

Reply:
exactly, we can do it, IIRC @Molter73 asked something similar some time ago